### ISSUE TITLE
'ModelOfferentityOffersCompositeKeyConditionInput' is not present when resolving type 

### ISSUE THREAD

**Issue by oliverandersencox** [aws-amplify/amplify-cli#2287]
Found 2 problem(s) with the schema: The field input type The field input type 'ModelOfferentityOffersCompositeKeyConditionInput' is not present when resolving type 'Query' [@867:1].

I have created a fresh amplify project and added auth and storage with no problems. 

When trying to push the api I get these errors:

> The field input type 'ModelOfferentityOffersCompositeKeyConditionInput' is not present when resolving type 'Query' [@867:1].

```
type Offer @model
  @key(name: "entityOffers", fields: ["entityId", "startDate", "endDate"], queryField: "offerByDate")
    @auth(rules: [
    { allow: owner, operations: [update, delete, create] },
    { allow: groups, groups: ["admin"], operations: [update, delete, create] }
  ])
 {
  id: ID!
  entityId: String!
  startDate: AWSDateTime!
  endDate: AWSDateTime!
  description: String
  price: Float
  image: String
  title: String!
  status: String!
  createdAt: Float!
  updatedAt: Float!
}
```

This is a fresh project so I cant understand why this is happening

**Comment by kaustavghosh06** [comment]
This is a duplicate of this issue #2239 
We have pushed a fix for this as a part of #2246 
As a workaround, if you change your @key(name: "entityOffers") to @key(name: "EntityOffers"), your push should work.

**Comment by kaustavghosh06** [comment]
We released the latest version of the CLI - 3.8.0 with this fix.
